src/test/java/duke/task/TaskTest.java

```diff
@@ -0,0 +1,46 @@
+package duke.task;
+
+import duke.DukeException;
+import duke.utils.TaskList;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class TaskTest {
+    @Test
+    public void markTaskDone_success() throws DukeException {
+        TaskList task = new TaskList();
```

Comment:
Would the plural form might be more appropriate for the naming of this variable since it is a collection of objects? I noticed the same issue in the other test methods as well.